core/pom.xml

```diff
@@ -34,6 +34,7 @@
         <dependency>
             <groupId>com.github.javaparser</groupId>
             <artifactId>javaparser-core</artifactId>
+            <version>3.5.0</version>
```

Comment:
Is there a reason for not leaving that managed in the parent? Also it seems there's a newer version (3.9.1)?

Reply:
Not really. The only reason was that the other shaded library (ASM) is also not managed in the parent. Should I put javaparser back in the parent?

The current code doesn't work with JavaParser 3.9.1. They made several incompatible API changes since 3.5.0, the first one between JavaParser versions 3.6.15 and 3.6.16. That's why I wanted this library shaded.

I plan to submit a separate PR with updated dependency versions (javaparser 3.9.1, ASM 7.0 etc.) and the necessary code adjustments.

Reply:
> Should I put javaparser back in the parent?

Yes, please. I'd prefer all versions to be managed in the parent POM, i.e. ASM was the outlier here. Thanks!

Reply:
Done.